### PR TITLE
Updated java file

### PR DESCRIPTION
**Pull request by AdityaShaw1**

### CODE REVIEW

**Comment by AdityaShaw1** [comment]
i have created a pull request kindly pls review it

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
Thank you @AdityaShaw1, could you please resolve merge conflicts before

File: java/java-quiz.md [hunk 1]
```diff
@@ -2103,16 +2103,62 @@ Queue<Rainfall>
...
-#### Q150. What do these statments evaluate to?
+- [ ] white
+      white
+      blue
 
-- [ ] 1. false 2. true
-- [ ] 1. false 2. false
-- [ ] 1. true 2. true
-- [x] 1. true 2. false
+#### Q 151. Which command will run a FrogSounds app that someone emailed to you as a jar?
```
Comment (auchynnikau) [inline java/java-quiz.md:2160]:
```suggestion
#### Q151. Which command will run a FrogSounds app that someone emailed to you as a jar?
```

**Review by auchynnikau — CHANGES_REQUESTED** [review]

**Comment by Ebazhanov** [comment]
@AdityaShaw1  ping

**Comment by AdityaShaw1** [comment]
I will complete it by tmrw morning pls give me sometime

**Comment by Ebazhanov** [comment]
no problem, take your time!

**Pull request closed without merging**